WundergroundNetConsole/Program.cs

```diff
@@ -5,29 +5,36 @@
 using System.Threading.Tasks;
 using WundergroundNetLib;
 
+using WundergroundNetLib.Interfaces;
+
 namespace WundergroundNetConsole
 {
     class Program
     {
         static void Main(string[] args)
         {
-            WundergroundDataProvider dataProvider = new WundergroundDataProvider();
+            IWundergroundDataProvider dataProvider = WundergroundDataProvider.DefaultProvider; // ensure only one instance is created using the WundergroundDataProvider singleton
+
             //var WundergroundData = dataProvider.GetWundergroundWeatherDataAsync("-43.506923", "172.731346");
             //var WundergroundData = dataProvider.GetWundergroundWeatherDataAsync("ICANTERB275");
             var WeatherData = dataProvider.GetWundergroundWeatherDataAsync(-43.506923, 172.731346);
 
-            Console.WriteLine();
-            Console.WriteLine("--------------------------------------");
-            Console.WriteLine("Observation Location Details");
-            Console.WriteLine();
-
-            Console.WriteLine("City \t\t\t{0}", WeatherData.Result.observationLocation.City);
-            Console.WriteLine("Country \t\t{0}", WeatherData.Result.observationLocation.Country);
-            Console.WriteLine("StationLatitude \t{0}", WeatherData.Result.observationLocation.StationLatitude);
-            Console.WriteLine("StationLongitude \t{0}", WeatherData.Result.observationLocation.StationLongitude);
-            Console.WriteLine("StationElevation \t{0}", WeatherData.Result.observationLocation.StationElevation);
-            Console.WriteLine("StationID \t\t{0}", WeatherData.Result.observationLocation.StationID);
-            Console.WriteLine("WmoNumber \t\t{0}", WeatherData.Result.observationLocation.WmoNumber);
+            Console.WriteLine(string.Format("--------------------------------------\r\n" +
```

Comment:
I've found this format easier to read, though it's personal preference so I didn't change them all. Feel free to ignore.